.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  release:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+    - name: Fetch source code
+      uses: actions/checkout@v2
+      with:
+        submodules: 'recursive'
+    - name: Use Nix
+      uses: cachix/install-nix-action@v12
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Configure Nix substituters
+      run: |
+        set -xe
+        mkdir -p ~/.config/nix/
+        cp ./.github/nix.conf ~/.config/nix/
+    - uses: cachix/cachix-action@v10
+      with:
+        name: perspect3vism
+        # authToken: '${{ secrets.CACHIX_HOLOCHAIN_CI }}'
+    - name: Prepare Nix environment
+      run: nix-shell --command "echo Completed"
+    - name: Release languages
+      run: ./release.sh
+    - name: Upload released files
+      uses: ncipollo/release-action@v1
+      with:
+        artifacts: "release/full_time_index/bundle.js, release/full_time_index/social-context.dna, release/full_index.zip, release/full_time_index.zip, release/signal.zip"
```

Comment:
I'm not sure whether this bundle.js is that we used in flux release? @jdeepee

Reply:
the bundle.js used for release is the one that is generated with the `full_index` configuration